src/askExpressMiddleware/askExpressMiddleware.spec.ts

```diff
@@ -0,0 +1,63 @@
+import { runUntyped } from '../askvm';
+import { askExpressMiddleware } from './askExpressMiddleware';
+import { NextFunction } from 'express';
+
+jest.mock('../askvm', () => {
+  return {
+    ...jest.requireActual('../askvm'),
+    runUntyped: jest.fn(),
+  };
+});
```

Comment:
Very nice! This pattern should be a utility. Even better that Jest enforces that the test and the VM run in different environments.